src/python/WMCore/WMBS/CreateWMBSBase.py

```diff
@@ -278,15 +278,15 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE TABLE wmbs_job (
              id           INTEGER       PRIMARY KEY AUTO_INCREMENT,
              jobgroup     INTEGER       NOT NULL,
-             name         VARCHAR(255),
+             name         VARCHAR(70),
              state        INTEGER       NOT NULL,
              state_time   INTEGER       NOT NULL,
              retry_count  INTEGER       DEFAULT 0,
              couch_record VARCHAR(255),
              location     INTEGER,
              outcome      INTEGER       DEFAULT 0,
-             cache_dir    VARCHAR(767)  DEFAULT 'None',
-             fwjr_path    VARCHAR(767),
+             cache_dir    VARCHAR(70)  DEFAULT 'None',
+             fwjr_path    VARCHAR(70),
```

Comment:
Alan, you're correct and I'll revert this once I'll figure out why MariaDB
can't handle unique names longer than 255. If anyone knows the fix please speak
up.

On  0, Alan Malta Rodrigues <notifications@github.com> wrote:
> >               state        INTEGER       NOT NULL,
> >               state_time   INTEGER       NOT NULL,
> >               retry_count  INTEGER       DEFAULT 0,
> >               couch_record VARCHAR(255),
> >               location     INTEGER,
> >               outcome      INTEGER       DEFAULT 0,
> > -             cache_dir    VARCHAR(767)  DEFAULT 'None',
> > -             fwjr_path    VARCHAR(767),
> > +             cache_dir    VARCHAR(70)  DEFAULT 'None',
> > +             fwjr_path    VARCHAR(70),
> 
> Valentin, to avoid forgetting that we do need to update these columns back to their original values before merging this PR, can you please update them now? It will probably be merged eventually, so better update the code now then let it go to production. Thanks
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6260/files#r39917446